test/lib/primer/forms/forms_test.rb

```diff
@@ -54,6 +55,51 @@ def test_the_input_is_described_by_the_caption
     assert_selector "input[aria-describedby='#{caption_id}']"
   end
 
+  def test_includes_activemodel_validation_messages
```

Comment:
I moved these to test/lib/primer/forms/form_helper_test.rb, so they can be removed from here.

I think we also want to `require "deep_thought"` in test/lib/primer/forms/form_helper_test.rb and remove the `DeepThought` class.